src/components/AxisLabel/AxisLabel.jsx

```diff
@@ -61,16 +72,25 @@ const AxisLabel = createClass({
 			orient,
 			label,
 			color,
+			style,
 			className,
 			...passThroughs,
 		} = this.props;
 
 		const isH = orient === 'top' || orient === 'bottom';
+		const isCustomColor = _.startsWith(color, '#');
+		const colorStyle = isCustomColor ? { fill: color } : null;
 
 		return (
 			<text
 				{...passThroughs}
-				className={cx(className, '&', `&-color-${color % 6}`)}
+				style={{
+					...style,
+					...colorStyle,
```

Comment:
i think the order here should be swapped...if the consumer wants to overwrite the colorStyle with their own `style`, they should be able to.